assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/styles.scss

```diff
@@ -48,10 +59,12 @@
 			}
 		}
 		td {
+			background-color: var(--ee-background-color);
+			height: 80px;
+
```

Comment:
done, because of the specificity, had to move things around:

![image](https://user-images.githubusercontent.com/1477453/81420310-79830280-9158-11ea-942f-0cdc366e4248.png)